dev/reference/tools/tidb-lightning/monitor.md

```diff
@@ -55,9 +55,117 @@ scrape_configs:
+
+### 进度面板
+
+![第二行进度面板](/media/lightning-grafana-row-2.png)
+
+第二行的面板显示进度，具体为：
+
+| 面板名称 | 描述 |
+|:-----|:-----|
+| Import progress | 已编码的文件所占百分比 |
+| Checksum progress | 已导入的表所占百分比 |
+| Failures | 导入失败的表的数量以及故障点，通常为空 |
+
+### 资源使用面板
+
+![第三行资源面板](/media/lightning-grafana-row-3.png)
+
+第三行的面板显示资源使用情况，具体为：
+
+| 面板名称 | 描述 |
+|:-----|:-----|
+| Memory usage | 每个服务占用的内存 |
+| Number of Lightning Goroutines | TiDB Lightning 正在使用的 goroutines 数量 |
+| CPU% | 每个服务使用的逻辑 CPU 数 |
+
+### 配额使用面板
+
+![第四行配额使用面板](/media/lightning-grafana-row-4.png)
+
+第四行的面板显示配额使用情况，具体为：
+
+| 面板名称 | 序列 | 描述 |
+|:-----|:-----|:-----|
+| Idle workers | io | 未使用的 `io-concurrency` 的数量，通常接近配置值（默认为5），接近 0 时表示磁盘太慢 |
+| Idle workers | closed-engine | 已关闭但未清理的引擎数量，通常接近 `index-concurrency` 与`table-concurrency` 的和（默认为 8），接近 0 时表示 TiDB Lightning 比 TiKV Importer 快，这会导致 TiDB Lightning 延迟 |
+| Idle workers | table | 未使用的 `table-concurrency` 的数量，通常为 0，直到进程结束 |
+| Idle workers | index | 未使用的 `index-concurrency` 的数量，通常为 0，直到进程结束 |
+| Idle workers | region | 未使用的 `region-concurrency` 的数量，通常为 0，直到进程结束 |
+| External resources | KV Encoder | 已激活的 KV encoder 的数量，通常与 `region-concurrency` 的数量相同，直到进程结束 |
+| External resources | Importer Engines | 打开的引擎文件数量，不应超过`max-open-engines` 的设置。 |
+
+### 读取速度面板
+
+![第五行读取速度面板](/media/lightning-grafana-row-5.png)
+
+第五行面板显示读取速度，具体为：
+
+| 面板 | 序列 | 描述 |
+|:-----|:-----|:-----|
+| Chunk parser read block duration | read block | 读取一个字节块以准备解析所消耗的时间 |
+| Chunk parser read block duration | apply worker | 等待 `io-concurrency` 空闲所消耗的时间 |
+| SQL process duration | row encode | 解析和编码单行所消耗的时间 |
+| SQL process duration | block deliver | 将一组键/值对发送到 TiKV Importer 所消耗的时间 |
+
+如果上述项的持续时间过长，则表示 TiDB Lightning 使用的磁盘太慢或 I/O 太忙。
+
+### 存储空间面板
+
+![第六行存储空间面板](/media/lightning-grafana-row-6.png)
+
+第六行面板显示存储空间使用情况，具体为：
+
+| 面板 | 序列 | 描述 |
+|:-----|:-----|:-----|
+| SQL process rate | data deliver rate | 向 TiKV Importer 发送键值对 data 的速度 |
+| SQL process rate | index deliver rate | 向 TiKV Importer 发送键值对 index 的速度 |
+| SQL process rate | total deliver rate | 键值对上传的总速度 |
+| Total bytes | parser read size | TiDB Lightning 正在读取的字节数 |
+| Total bytes | data deliver size | 已发送到 TiKV Importer 的键值对 data 的字节数 |
+| Total bytes | index deliver size | 已发送到 TiKV Importer 的键值对 index 的字节数 |
+| Total bytes | storage_size / 3 | TiKV 集群的占用大小的三分之一（3 为默认的副本数量）|
+
+### 导入速度面板
+
+![第七行导入速度面板](/media/lightning-grafana-row-7.png)
+
+第七行面板显示导入速度，具体为：
+
+| 面板 | 序列 | 描述 |
+|:-----|:-----|:-----|
+| Delivery duration | Range delivery | 将一系列键值对上传到 TiKV 集群所消耗的时间  |
+| Delivery duration | SST delivery | 将单个 SST 文件上传到 TiKV 集群所消耗的时间 |
+| SST process duration | Split SST | 将键值对流切分成若干 SST 文件所消耗的时间 |
+| SST process duration | SST upload | 上传单个 SST 文件所消耗的时间 |
+| SST process duration | SST ingest | Ingest 单个 SST 文件所消耗的时间 |
+| SST process duration | SST size | 单个 SST 文件的大小 |
+
 ## 监控指标
 
-本节将详细描述 `tikv-importer` 和 `tidb-lightning` 的监控指标。
+本节中将详细描述 Grafana 仪表盘支持的其他 `tikv-importer` 和 `tidb-lightning` 的监控指标。
```

Comment:
This section is not directly related to Grafana (the frontend) anymore. These are metrics for Prometheus (the backend)